### ISSUE TITLE
Move `TvId` in constructor in `TvShowDetailsUseCase`

### ISSUE THREAD

**Issue by Kirchhoff-** [Kirchhoff-/Movies#744]

**Comment by Kirchhoff-** [comment]
It turns out that we can't implement this task with our current architecture and the Koin library.

The problem is that:

> "parameters are provided at the time of resolving a dependency, not during its registration."


This means we have no way to pass a parameter into a module and then create all dependent objects afterward — as we could in Kodein or with manual DI. 

This limitation contradicts our overall architectural vision.We'll create a separate task to investigate possible changes to our DI approach and block this task until that is resolved.

**Comment by Kirchhoff-** [comment]
Blocked by - [#838](https://github.com/Kirchhoff-/Movies/issues/838)
